app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.all
+  end
```

Comment:
- [OPTIONAL] As you did for the user show controller on line 7 ` @user = User.find(params[:id])` let's make sure the show post action controller return one post. You can use `find and params` instead of the `.all` method. 😉  I made it optional but I strongly recommend you address this 💯 👍🏻

Reply:
Yeah, thanks you very much, i will fix that